From 6d48ab7c79896ab51543a5ffeb65ddf294051eb3 Mon Sep 17 00:00:00 2001
From: Jens Reidel <adrian@travitia.xyz>
Date: Mon, 22 Aug 2022 15:37:09 +0300
Subject: [PATCH] Disable Bellis and Firewall

Bellis is not used by any of the Calyx users for davinci and Firewall
functionality is provided in the app settings system-wide.

Change-Id: Ibc19ac2c889303e297d644e15ad81025e3a0d32c
---
 config/common.mk | 2 --
 1 file changed, 2 deletions(-)

diff --git a/config/common.mk b/config/common.mk
index 15ed20b..2c4d095 100644
--- a/config/common.mk
+++ b/config/common.mk
@@ -77,13 +77,11 @@ PRODUCT_PACKAGES += \
 PRODUCT_PACKAGES += \
     AudioFX \
     AuroraStorePrivilegedExtension \
-    Bellis \
     Etar \
     ExactCalculator \
     F-Droid \
     F-DroidPrivilegedExtension \
     Eleven \
-    Firewall \
     Recorder \
     Seedvault \
     Ripple \
-- 
2.37.2

